src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, "
+                + "radius: " + radius + "units, "
+                + "color:" + getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
```

Comment:
The calculation of the area of a circle is a magic number in your code. According to the checklist, all magic numbers in your code should be constants. Please see this article to learn about constant fields and their naming requirements: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names. You should create a constant for the number 2 that is used in the calculation.